src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -1980,13 +1980,13 @@ public void testCaseLevel() throws Exception {
         checkConfig.addProperty("arrayInitIndent", "4");
         checkConfig.addProperty("basicOffset", "4");
         checkConfig.addProperty("braceAdjustment", "0");
-        checkConfig.addProperty("caseIndent", "0");
+        checkConfig.addProperty("caseIndent", "4");
```

Comment:
Not related to the main issue, just saw that test is out of sync with test definition comment:
```
 * caseIndent = 4  
```
in [InputIndentationCaseLevel.java](https://github.com/checkstyle/checkstyle/blob/a42a3e1733f3c7583177a3c96de65e6a2fd7671f/src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationCaseLevel.java#L9) (given that we have a separate test for 0 case indent [testZeroCaseLevel()](https://github.com/checkstyle/checkstyle/blob/a42a3e1733f3c7583177a3c96de65e6a2fd7671f/src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java#L243) it seemed more logical to test indent 4 here)

Reply:
Thanks for finding this, please just update the config in the test input to match property in unit test as a separate commit: `minor: update test input config to reflect unit test config`.

Reply:
Done (updated test input javadoc to match config in JUnit test and extracted this change into separate commit)